src/js/smooch.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { render, unmountComponentAtNode } from 'react-dom';
 import pick from 'lodash.pick';
+import { batchActions } from 'redux-batched-actions';
 
 import { store } from './stores/app-store';
```

Comment:
Using webpack code splitting, requiring a module becomes a promise so you just need to chain it.